src/Native/Unix/Common/pal_utilities.h

```diff
@@ -148,6 +148,8 @@ inline void SafeStringCopy(char* destination, int32_t destinationSize, const cha
     }
 }
 
+#else // __cplusplus
+#define ARRAY_SIZE(a) (sizeof(a)/sizeof(a[0]))
```

Comment:
Is there any risk this gets used with an empty array?

Reply:
I've just tested:

```sh
clang -x c -o test.exe - << EOF
#define ARRAY_SIZE(a) (sizeof(a)/sizeof(a[0]))
#include <stdio.h>
int main(void)
{
  static char first[] = {};
  static char second[] = "";
  static char third[] = "\0";
  printf("count of first %lu\n", ARRAY_SIZE(first));
  printf("count of second %lu\n", ARRAY_SIZE(second));
  printf("count of third %lu\n", ARRAY_SIZE(third));
  return 0;
}
EOF
```

produces:

count of first 0
count of second 1
count of third 2

Reply:
Ok. Thanks.